wrappers/C#_High_Level_Wrapper/RefpropDLL/Refprop_DLL.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace RefpropDLL
+{
+    public class Refprop_DLL
+    {
+        [DllImport(@"REFPROP.DLL")]

```

Comment:
This implies 32-bit, not what you want

Reply:
Yes, I can do it for 64 bit too. If you require.

Reply:
Yes, 32-bit is obsolete. Better to do only 64-bit if you are only going to do one.